llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -29773,36 +29773,35 @@ SDValue AArch64TargetLowering::LowerFixedLengthVECTOR_SHUFFLEToSVE(
     return convertFromScalableVector(DAG, VT, Op);
   }
 
+  auto lowerToRevMergePassthru = [&](unsigned Opcode, SDValue Vec,
+                                     EVT PredVecVT, EVT RevVT) {
```

Comment:
I don't like the lambda here because there's an untidy mix of implicit and explicit parameters but above all it makes the second use overly complicated because it does not need the bitcasts.  This will be further compounded if/when we decide to fix the predicate used for the non-revd case.

Reply:
:+1: I've inlined the lambda (and cleaned up some naming).